task3.py

```diff
@@ -29,22 +29,18 @@
     print('To stop entering boys\' names type "stop"!')
     boy_name = str()
     boys = set()
-    while True:
+    while boy_name.lower() == 'stop':
```

Comment:
Looks like I did something wrong... Doesn't work the correct way. I didn't quite understand what I had to do

Reply:
Of course you did! The condition is not correct. Look at it closely: you enter the loop (and the loop works) only if boy's name is equal to "stop". But it is not what we need. We need it to be reversed. And then it will start working absolutely correct!